definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -361,7 +361,7 @@
     "CaseTypeID": "DIVORCE",
     "CaseFieldID": "generalApplicationWithoutNoticeFeeSummary",
```

Comment:
I think it might be only that field that needs "CRUD"

Reply:
I think it's because we update it from COS.
Maybe because it's a complex type, it has to remove it before replacing it?